lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -169,7 +169,7 @@ defmodule ExUnit.Case do
   messages generated while running a test are captured and only if the test fails are they printed
   to aid with debugging.
 
-  You can opt into this behavior for individual tests by tagging them with `:capture_log` or enable
+  You can opt into this behaviour for individual tests by tagging them with `:capture_log` or enable
```

Comment:
Can we please keep only this change? Let's change only entries in the Elixir source code.